services/121-service/src/main.ts

```diff
@@ -69,6 +70,7 @@ function generateModuleDependencyGraph(app: INestApplication): void {
 }
 
 async function bootstrap(): Promise<void> {
+  IntersolveVisaStatusMapper.loadMapping(); // TODO: change where this is loaded
```

Comment:
I think this should be removed right?

Reply:
YEs, I removed it. But did you test the code change with the static isMappingLoaded also with this code in there? If so, was this then tested correctly?